clients/java/src/main/java/io/camunda/client/api/response/UserTaskEvent.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.client.api.response;
+
+import java.util.List;
+import java.util.Map;
+
+public interface UserTaskEvent {
```

Comment:
🔧 No need to create this explicitly. The `...Event` interfaces are usually created for Zeebe interactions with the API, e.g. creating resources like a process instance. Since we don't do this for user tasks, there is no need for this. You can directly keep all this in the `UserTask` resource defined as a search response.

💭 Should we need to implement this in the future, e.g. for handling user task listeners in the client, we can create it and have the `UserTask` interface inherit from the then-created event class. But we don't need this now.

ℹ️ If you'd still like to keep this event class already, it should only contain the runtime data provided by Zeebe. Things like the `state` and `completionDate` are not provided by Zeebe for user tasks. Those two would have be removed here. The `UserTask` search response interface could add them on top of this interface here then.